server/__tests__/test.js

```diff
@@ -0,0 +1,22 @@
+const server = require("../server")
+const supertest = require("supertest");
+const mongoose = require("mongoose");
+require("dotenv").config();
+
+const address = process.env.MONGODB_URL_TEST || "mongodb://localhost:27017/test?directConnection=true&serverSelectionTimeoutMS=2000"
+
+beforeEach(async () => {
+    await mongoose.connect(address);
+});
+
+afterEach(async () => {
+    await mongoose.connection.close();
+});
+
+describe('test version', () => {
+    it('should return the version information', async () => {
+        const res = await supertest(server).get('/');
+        expect(res.status).toEqual(200);
+        expect(res.body).toEqual({ "version": "V1" });
```

Comment:
This is where the tests are failing for the server. I think you might have rebased an old version of main, so you can either just revert the changes made to this file or pull main & rebase it again.

Reply:
That is correct yeah, this is a very old test we had a while ago. This file is now at api.test.js, so there is no need for it, it is exactly the same minus the "V2" change below.

Pulling main again should fix these, including the new .gitignore that should not have included the package-lock.json file too.

Reply:
fixed by rebase